test/daal-fry/recipeTest.txt

```diff
@@ -0,0 +1,2 @@
+1. Look: Check the consistency and color of the dal.
```

Comment:
It is daal and not dal.

Reply:
The spelling dal -> daal has been replaced at both the places